applications/GeoMechanicsApplication/tests/shell_with_reset_displacement/MeshStructure.svg

```diff
@@ -0,0 +1,366 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Nice solution for the picture. Maybe remove the "m". I intended mm, but this is units in = units out.

Reply:
Yes, but I think it's still a good idea to have sensible things in here, so I changed all units to mm to clarify the initial intention of scale.